cloud/billing/account-management.md

```diff
@@ -1,37 +1,57 @@
 # Billing - Account management
 If you haven't yet added a payment method, you can add one in this section. You
-can also update to a new credit card here.
+can also update to a new credit card here. 
 
+## Add, update, or delete a payment method
+You can add up to three credit cards to the `Wallet` and set one of the cards as
+`primary` for payments. If you want to delete the last card in the `Wallet`, you
+have to first delete all the services and then schedule the last card to be
+deleted at the end of the month.
```

Comment:
Just a note: this is the case for the paying customers. If a user is still on trial he/she can remove the CC without the need to delete the services.